base/fastmath.jl

```diff
@@ -124,6 +125,38 @@ function make_fastmath(symb::Symbol)
 end
 make_fastmath(expr) = expr
 
+"""
+@fastmath expr
```

Comment:
Note that `make docs` is only needed for testing. There shouldn't be anything that you need to push after that.